src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/AbstractEntityAIGuard.java

```diff
@@ -88,6 +89,54 @@
      */
     private BlockPos currentPatrolPoint = null;
```

Comment:
### Errors:
- Field initialization to null is redundant 

### Warnings:
- Instance field name currentPatrolPoint doesn't match regex 'm_[a-z][A-Za-z\d]*'